docs/glues/herbs2rest.md

```diff
@@ -27,46 +27,65 @@ The method needs a list of controllers like the example below:
 const controllerList = [
   {
     name: 'lists',
-    idParameter: 'listId',
-    getAll: require('../usecases/getLists'),
-    getById: require('../usecases/getLists'),
-    post: require('../usecases/createList'),
-    put: require('../usecases/updateList'),
-    delete: require('../usecases/deleteList')
+    getAll: { usecase: require('../usecases/getLists'), controller: require('../controller') },
+    getById: { usecase: require('../usecases/getLists'), id: 'listId' },
+    post: { usecase: require('../usecases/createList') },
+    put: { usecase: require('../usecases/updateList') },
+    delete: { usecase: require('../usecases/deleteList') }
   }
 ]
 ```
 
 The `name` field is the name of the route (Ex. *https://example.com/lists*)
 
-The `idParameter` field is the param's custom name of the route (GetById, Put and Delete), if the field does not exist, the parameter name will be "*id*".
+The `id` field is the param's custom name of the route (GetById, Put and Delete), if the field does not exist, the parameter name will be "*id*".
+
+The controller field is to replace the default controller.
 
 The other fields refer to http methods using usecases (GetAll, GetById, Post, Put and Delete).
 
+#### Custom Controller
+
+To create a custom controller, it is necessary to follow this pattern.
+
+```javascript
+const controller = async (usecase, req, user, res, next) => {
+  // Implementation
+}
+```
+
+Each method parameter has different data:
+
+- usecase: usecase in ([buchu](https://github.com/herbsjs/buchu)) pattern.
+- req: body, query and params of route.
+- user: parameter passed in the request.
+- res: response object of [express](https://expressjs.com/).
+- next: allows the next queued route handler/middleware to handle the request.
+
 #### Generate Routes
 
 Generating and using new express routes:
 
 ```javascript
 const express = require('express')
-const { generateRoutes } = require('herbs2rest')
+const { generateRoutes } = require('@herbsjs/herbs2rest')
 
 const app = express()
 const routes = new express.Router()
 
-generateRoutes(controllerList, routes)
+generateRoutes(controllerList, routes, true)  // true = console info endpoints
 
 app.use(routes)
 ```
 
 #### Authorization
 
-All usecases need to implement the `authorize` method and receive a user for authentication..
+All use cases must implement the `authorize` method and receive a user for authentication if using the default controller.
 
 Example:
 
 ```javascript
-const { Ok, Err, usecase } = require('buchu')
+const { Ok, Err, usecase } = require('@herbsjs/buchu')
```

Comment:
here, You dont need to use buchu anymore. Just use **@herbsjs/herbs**

Reply:
ok, resolved